Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt-get update && \
     apt-get install --yes \
         build-essential ruby-dev ruby-bundler python3-dev python3-pip \
         libpq-dev libjpeg-dev zlib1g-dev libpng12-dev libmagickwand-dev \
-        libjpeg-progs optipng git vim curl
+        libjpeg-progs optipng git vim curl jq python-launchpadlib
```

Comment:
`python-launchpadlib` seems like a package more suited to an applications python requirements. Is there a reason it wouldn't be isntalled through pip?

Reply:
I realise looking through the script you linked that this simplifies the problem. Though if you add a requirements.txt to the codebase, ./run could install the package from that.

Reply:
LaunchpadLib is a helper tool for the local development helper script, and not a dependency needed to actually run the project, so I'm not sure `requirements.txt` is quite the right place for it.

We do install other python libraries as helpers for local development - `ipdb` etc. So I think it's fine to install launchpadlib in this way.

Reply:
Yeah, I am happy to go ahead

Reply:
It seems like poor coupling to put it here.  In particular it means that if that script switches to python3 then things will stop working.

Is there no way to arrange for the dependencies to live in the individual project in question, rather than in this more general repository?

Reply:
It is particularly important to keep dependencies which are built into the production image with the project, and we always do.

But all projects rely on some things existing in the system (build.snapcraft.io more than most). And launchpadlib is not a dependency of the build.snapcraft.io application, only of the local development tooling.

In this image we instead keep dependencies which are *purely* to support local development and are generally useful - like `ipdb` and `flake8`. This is less generally useful than those two, but it is purely for local development and I think it may well be useful to another of our applications.

At the moment, when we have `requirements.txt` in projects they are for dependencies of the production application, not for local development tooling. We could in the future introduce a standard way of including development dependencies for a project, and if we do we could consider whether this is a generally useful tool that should always be available or something that should be moved to the build.snapract.io project itself.

For now, however, let's leave it here.